app/src/main/java/org/mifos/mobile/ui/fragments/ReviewLoanApplicationFragment.kt

```diff
@@ -83,16 +71,14 @@ class ReviewLoanApplicationFragment : BaseFragment() {
         }
     }
 
-
-    private lateinit var viewModel: ReviewLoanApplicationViewModel
+    private val viewModel by activityViewModels<ReviewLoanApplicationViewModel>()
```

Comment:
use this `private val viewModel: ReviewLoanApplicationViewModel by viewModels()` as we are working with single fragment and we need viewmodel scoped to this fragment. We can use your declaration when you want to share ViewModel between fragments within the same activity.

Reply:
Yes now I changed it to viewModels<ReviewLoanApplicationViewModel>.